toolbox/xnvme_libconf.py

```diff
@@ -171,13 +172,40 @@ def update(args):
         ver += '\t"conf: %s",\n' % flag
         ver += "#endif\n"
 
+    ver += """
+#ifdef NVME_IOCTL_IO64_CMD
+    "3p: NVME_IOCTL_IO64_CMD",
+#endif
+#ifdef NVME_IOCTL_IO64_CMD_VEC
+    "3p: NVME_IOCTL_IO64_CMD_VEC",
+#endif
+#ifdef NVME_IOCTL_ADMIN64_CMD
+    "3p: NVME_IOCTL_ADMIN64_CMD",
+#endif
+    """
+
+    """
```

Comment:
This `"""` looks to be here by mistake.

Reply:
Ah, I see this manually coded in `xnvme_libconf.c` - what is the reason for this?